gather_stats.js

```diff
@@ -14,11 +14,8 @@ const yargs = require('yargs/yargs');
 {
     const result = dotenv.config();
 
-    if (result.error) {
-        throw result.error;

```

Comment:
PAT is set via environment variable by GitHub action. Wouldn't want to block it if the `.env` file does not exist